guide/C/ch_basics.xml

```diff
@@ -619,11 +619,151 @@
+				<application>&app;</application> does not currently implement these features, 
+				although it is a long term goal of the development team. </para>
+			</note>
+
+            <table frame="topbot" id="storage-comparison">
+              <title>Storage Comparison</title>
+
+              <tgroup align="left" cols="5">
+                <colspec colname="c1item" ></colspec>
+                <colspec colname="c2xml" ></colspec>
+                <colspec colname="c3sqlite" ></colspec>
+                <colspec colname="c4mysql" ></colspec>
+                <colspec colname="c5postgresql" ></colspec>
+
+                <thead>
+                  <row>
+                    <entry> </entry>
+                    <entry align="center">XML</entry>
+                    <entry align="center">SQLite</entry>
+                    <entry align="center">MySQL</entry>
+                    <entry align="center">PostgreSQL</entry>
+                  </row>
+                </thead>
+
+                <tbody>
+                <row>
+                  <entry >Installation</entry>
+  				<entry align="center" namest="c2xml"        > Default </entry>
+  				<entry align="center" namest="c3sqlite"     > libdbi - 1 </entry>
+  				<entry align="center" namest="c4mysql"      > libdbi </entry>
+  				<entry align="center" namest="c5postgresql" > libdbi </entry>
+                </row>
+
+                <row>
+                  <entry >File extension</entry>
+  				<entry align="center" namest="c2xml"        > gnucash </entry>
+  				<entry align="center" namest="c3sqlite"     > gnucash </entry>
+  				<entry align="center" namest="c4mysql"      > N/A - 2 </entry>
+  				<entry align="center" namest="c5postgresql" > N/A - 2 </entry>
+                </row>
+
+                <row>
+                  <entry >Additional software</entry>
+  				<entry align="center" namest="c2xml"        > None </entry>
+  				<entry align="center" namest="c3sqlite"     > None </entry>
+  				<entry align="center" namest="c4mysql"      > MySQL </entry>
+  				<entry align="center" namest="c5postgresql" > PostgreSQL </entry>
+                </row>
+
+                <row>
+                  <entry >Additional expertise</entry>
+  				<entry align="center" namest="c2xml"        > None </entry>
+  				<entry align="center" namest="c3sqlite"     > None </entry>
+  				<entry align="center" namest="c4mysql"      > DBMS </entry>
+  				<entry align="center" namest="c5postgresql" > DBMS </entry>
+                </row>
+
+                <row>
+                  <entry >Compression</entry>
+  				<entry align="center" namest="c2xml"        > Y </entry>
+  				<entry align="center" namest="c3sqlite"     > N </entry>
+  				<entry align="center" namest="c4mysql"      > N </entry>
+  				<entry align="center" namest="c5postgresql" > N </entry>
+               </row>
+
+                <row>
+                  <entry >Save on command</entry>
```

Comment:
"Save on request" or "Save on demand" ?

However more generally I would avoid simple "Y/N" cells, especially if you have two lines essentially about the same topic. How about combining them into one line in something like
```
<entry >Save strategy</entry>
<entry align="center" namest="c2xml"                                          > On demand </entry>
<entry align="center" namest="csqllite" namend="c5postgresql" > On commit </entry>
```